database/factories/QuestionFactory.php

```diff
@@ -7,8 +7,6 @@
 
 $factory->define(Question::class, function (Faker $faker) {
     return [
-        'source_url' => "https://placeimg.com/640/480/any",
-        'type' => $faker->randomElement(['photo', 'video', 'audio']),
         'max_score' => $faker->numberBetween(10, 50),
         'level' => $faker->unique()->numberBetween(1, 50),
```

Comment:
use randomnumber for level
`$faker->unique()->randomNumber()`

Reply:
I think numberBetween() is fine because randomNumber() generates a number of fixed width but we want to generate levels 1 to 20